src/navigator-card-body.jsx

```diff
@@ -370,7 +370,7 @@ export const NavigatorCardBody = ({
     }
 };
 
-function Item({ file, selected, setSelected, onDoubleClickNavigate, isGrid }) {
+const Item = React.memo(function Item({ file, isSelected, setSelected, onDoubleClickNavigate, isGrid }) {
```

Comment:
Uhhhh, this is a function component, given the same input it should not re-render, why are we using memo here?

Reply:
Except when testing it does not.... that's strange. Also note that selecting an item re-renders an `Item` twice for some reason :thinking:

Reply:
All components re-render whenever the parent component re-renders. I think there's confusing between the terms "re-render" and "updating the dom". "re-render"s always happen for everything. "updading the dom" happens only when there's different out from the component.